compile.ps1

```diff
@@ -0,0 +1,5 @@
+#Prerequisite to be ran in a shell with VS2017 dev tools environment loaded
```

Comment:
Can you please rename the script in Build.ps1

Reply:
Sure.